eng/ci/host/official-release.yml

```diff
@@ -0,0 +1,152 @@
+parameters:
+- name: publishToProduction
+  displayName: Publish to production?
+  type: boolean
+  default: false
+
+pr: none
+trigger: none
+
+resources:
+  repositories:
+  - repository: 1es
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+  - repository: eng
+    type: git
+    name: engineering
+    ref: refs/tags/release
+  pipelines:
+  - pipeline: build
+    source: dotnet-host.official
+
+variables:
+- template: ci/variables/cfs.yml@eng
+- name: artifact_name
+  value: NugetPackages
+  readonly: true
+- name: drop_path
+  value: $(Pipeline.Workspace)/build/${{ parameters.artifactName }}
+  readonly: true
+- name: staging_feed
+  value: public/infra-pre-release
```

Comment:
These names are not final; I have not created the feeds yet. Want to discuss them here or over teams.